theme/cerveza/src/components/_Dialog.scss

```diff
@@ -0,0 +1,106 @@
+.brew-Dialog {
+    display: none;
+    align-items: center;
+    justify-content: center;
+    overflow: hidden;
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    z-index: 40;
+
+    .brew-Dialog {
+
+        &__background {
+            background-color:  rgba(0, 33, 48, 0.9);
+            position: absolute;
+            left: 0;
+            top: 0;
+            bottom: 0;
+            right: 0;
+        }
+
+        &__content {
+            z-index: 1;
+            width: 100%;
+            max-width: 530px;
```

Comment:
Likewise, should this value be derived from existing sizes?

Reply:
@ItsEricWoodward The question I used to guide this decision is: "Would I want this to change if my base size changed?".  I don't think so, so I made it a static size.  It also has this static size in the design docs.

Reply:
@joshrcook I understand where you're coming from, but I think that we should plan on (or, at least, allow for) *everything* to change size when the font does - that's the "responsive" in responsive design. Doing so helps us minimize the work we need to do in the future (if they decide to put OvrC on a TV screen, for example) without it being a premature optimization (530px / base size of 16px = approx 33rem, so we can write it once as `size(normal) * 33` and not worry about it going forward).

Incidentally, even if we don't make it a responsive width, I still feel that we shouldn't have explicit PX widths defined anywhere but in our variables (with a handful of exceptions, like 0px or 1px for borders, padding, etc). What I mean is, if this component is 530px, it's a pretty good bet that some other component will be the same size, and we should use the same variable for both so that we have consistency across components. That way, if someone decides tomorrow that everything 530px across should actually be 545px, we won't have to search for all of them and manually change them.

I might even suggest combining this with responsive widths - instead of it being 33rem (`size(normal) * 33`), the modifier could also be defined as a variable (ex: `size(normal) * map-get($medium, boxes)` or somesuch), so when it gets changed to 545px (or `size(normal) * 34`), again, we'd only have to update the modifying variable.

Maybe it would be best to wait for tomorrow's standup and discuss this further as a team?